app/scripts/components/rules/edit.js

```diff
@@ -19,14 +19,15 @@ const EditRule = React.createClass({
   },
 
   render: function () {
-    const { ruleName } = this.props.params;
+    const { params, rules } = this.props;
```

Comment:
It doesn't seem like you're using the value of `params` anywhere? If so, you can remove it

Reply:
I'm using params in the next line (23) to pull out the ruleName.

Reply:
Not sure how I missed that. My bad